src/main/java/cn/nukkit/Player.java

```diff
@@ -2495,15 +2495,6 @@ protected void processLogin() {
             return;
         }
 
-        for (Player p : new ArrayList<>(this.server.playerList.values())) {

```

Comment:
I think it's best not to modify the checking position of players with the same name